src/color-system.mdx

```diff
@@ -0,0 +1,98 @@
+
+### Dark mode, light mode, usability
+**Note:** don't confuse titles like "dark mode" and "light mode" or "dark/light themes" with these swatches' naming convention. As mentioned above, each swatch -despite its name- already comes with two tones inbuilt into the color system. For example the `--color-red-dark` variable will automatically be displayed as `#d32f2f;` in light mode, and as `#c62828` in dark mode.
+
+<limel-example-dark-light-mode />
+
+The reduced brightness in the dark mode is set with usability in mind, and will help reduce users' eye strain in low-light environments. In a proper design for dark mode, it is not enough to invert the background and text colors. Colors which are bright in light mode, will appear even brighter in dark mode and therefore pop out more from the UI, which can create visual nuisance.
+
+
+## Usage
+### Using CSS variables
+We have not used HEX values to create the CSS variables for each color. They are instead written in RGB digits, without parentheses. For example `--color-blue-default: 33, 150, 243;` (which would be `#2196f3` in HEX). The reason for such a choice is empowering you to write the colors with alpha channel if needed.
+
+Thus to get a solid blue color, you would write this: `color: rgb(var(--color-blue-default));`,
+or like this to get a blue color with 10% opacity: `color: rgba(var(--color-blue-default), 0.1);`
+
+### Available palettes
+In your web-components, you only need to import one of the palettes. Importing a palette makes all included CSS variables available in that web-component. It all depends on how many colors you need for that specific component. If you need more colors than what is offered in the UI palette, then simply remove that and import one of the other palettes that suits.
+
+Each palette also includes Lime Technologies' brand colors. So no matter which one you import, you will always have access to our brand colors and can use them by their specific CSS variable names.
+
+#### **UI palette** (default)
+Includes 10 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+
+<limel-example-ui-color-palette />
+
+In most cases, importing the UI palette is enough and gives you colors which you normally need in a user interface. This palette includes a few hand picked swatches that add an extra layer of meaning to different elements. Using these, an element in the UI can quickly communicate to the user what it does or indicates.
+
+As you notice, for each hue there is a `-default` and `-light` swatch available in this palette. The `-light` tints are usually good to use when creating interactive visual feedback. For example a button that has `--color-green-default` as background color, can get `--color-green-light` when it is hovered by the cursor.
```

Comment:
I'd suggest this structure for the first sentence:

```suggestion
In this palette, there is a `-default` and `-light` swatch available for each hue. The `-light` tints are usually good to use when creating interactive visual feedback. For example a button that has `--color-green-default` as background color, can get `--color-green-light` when it is hovered by the cursor.
```

I'd also suggest either *"there is a `-default` and a `-light` swatch available"*, or *"there are `-default` and `-light` swatches available"*:

```suggestion
In this palette, there is a `-default` and a `-light` swatch available for each hue. The `-light` tints are usually good to use when creating interactive visual feedback. For example a button that has `--color-green-default` as background color, can get `--color-green-light` when it is hovered by the cursor.
```

```suggestion
In this palette, there are `-default` and `-light` swatches available for each hue. The `-light` tints are usually good to use when creating interactive visual feedback. For example a button that has `--color-green-default` as background color, can get `--color-green-light` when it is hovered by the cursor.
```